README.rst

```diff
@@ -652,9 +707,257 @@ any WSGI server, such as uWSGI or Gunicorn. For example:
 
 .. code:: bash
 
-    $ pip install gunicorn
+    $ pip install requests gunicorn
     $ gunicorn things:app
 
+On Windows where Gunicorn and uWSGI don't work yet you can use Waitress server
```

Comment:
Just a thought, maybe here we could mention wsl for windows as an alternative?

Reply:
Not a bad idea... have you tried Gunicorn and uWSGI under WSL? Just want to sanity-check that it works. :)

Reply:
Updated the text.

Reply:
Tried just to be sure, both work correctly under wsl, and since wsl2 is practically a linux vm, I think we are safe to assume that wsl is an alternative :)

Reply:
just a nitpick. I've tried also uwsgi and it too works on wsl